tests/parametric/test_dynamic_configuration.py

```diff
@@ -801,3 +820,80 @@ def test_trace_sampling_rules_with_tags(self, test_agent, test_library):
         span = trace[0]
         assert "_dd.p.dm" in span["meta"]
         assert span["meta"]["_dd.p.dm"] == "-12"
+
+    @parametrize("library_env", [{**DEFAULT_ENVVARS}])
+    def test_remote_sampling_rules_telemetry(self, library_env, test_agent, test_library):
```

Comment:
[Question] There's already a [sampling rules test](https://github.com/DataDog/system-tests/blob/cdf9842939058ad28a98a8115c4e9048cef06a76/tests/parametric/test_dynamic_configuration.py#L572-L634). Is there a reason not to modify the test with telemetry assertions?

Reply:
I thought it would be a good separation of concerns, and since I made `set_and_wait_rc_telemetry` to set sampling rules, we can't use `set_and_wait_rc` anyways. But I can see it could be a very short test if all the libraries send the same payload and if we could reuse `set_and_wait_rc`.